scss/base/_reset.scss

```
@@ -73,3 +73,26 @@ p {
   margin: 0px;
   text-align: justify;
 }
+
+[style*="--aspect-ratio"] > :first-child {
+  width: 100%;
+}
+[style*="--aspect-ratio"] > img {
+  height: auto;
+}
+@supports (--custom: property) {
+  [style*="--aspect-ratio"] {
+    position: relative;
+  }
+  [style*="--aspect-ratio"]::before {
```

```suggestion
}

[style*="--aspect-ratio"] > img {
  height: auto;
}

@supports (--custom: property) {
  [style*="--aspect-ratio"] {
    position: relative;
  }
  
  [style*="--aspect-ratio"]::before {
```
Mỗi block cách nhau 1 dòng nhé